hnn_core/cell.py

```diff
@@ -14,6 +14,41 @@
 # Units for gbar: S/cm^2
 
 
+class ArtificialCell:
+    """The ArtificialCell class for initializing NEURON feed sources.
+    Parameters
+    ----------
+    event_times : list
+        Spike times associated with a given gid
```

Comment:
where is the gid?

Reply:
The gid does not need to specified/known within the scope of this instance, however, I thought that the user should be aware that each `ArtificialCell` instance is created for a specific gid in the `Network` class.